addons/account/models/product.py

```diff
@@ -52,3 +52,77 @@ def get_product_accounts(self, fiscal_pos=None):
         if not fiscal_pos:
             fiscal_pos = self.env['account.fiscal.position']
         return fiscal_pos.map_accounts(accounts)
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.model
+    def _get_default_price_unit(self, company, currency, document_date, document_type,
```

Comment:
```suggestion
    def _get_tax_included_unit_price(self, company, currency, document_date, document_type,
```

would be clearer IMHO

Reply:
agreed